### DISCUSSION TITLE
Problem while creating a composed chart

### DISCUSSION THREAD

**Discussion by Aatif-Thinkitive** [patrykandpatrick/vico#330] [Questions] [answered]
i want to create a graph like this as you can see this is a composedchart with three bar graphs and one linechart I am facing with composedchartentrymodelproducer when i use chart = remember(columnChart, lineChart) { columnChart + lineChart },
its not working as expected i am getting something like this 
![image](https://github.com/patrykandpatrick/vico/assets/108254342/8e7b0b72-4fe0-4a0e-bccc-30bd2991174e)

i want it like this 
![Screenshot from 2023-08-03 12-59-38](https://github.com/patrykandpatrick/vico/assets/108254342/0204451b-1edc-44da-b153-7e7c18873aba)

**Comment by patrickmichalik** [comment, marked as answer]
Hello! The fourth chart in the sample app matches the second image. Here’s an example with static data:

```kotlin
val chartEntryModel =
    entryModelOf(entriesOf(1f, 2f, 4f), entriesOf(2f, 1f, 4f), entriesOf(4f, 2f, 1f)) + entryModelOf(1f, 4f, 2f)

@Preview
@Composable
fun Example() {
    val columnChart = columnChart()
    val lineChart = lineChart()
    Chart(
        chart = remember { columnChart + lineChart },
        model = chartEntryModel,
        startAxis = startAxis(),
        bottomAxis = bottomAxis(),
    )
}
```
